sdk/identity/identity/rollup.base.config.js

```diff
@@ -2,7 +2,7 @@ import nodeResolve from "rollup-plugin-node-resolve";
 import multiEntry from "rollup-plugin-multi-entry";
 import cjs from "rollup-plugin-commonjs";
 import replace from "rollup-plugin-replace";
-import { uglify } from "rollup-plugin-uglify";
+import { terser } from "rollup-plugin-terser";
```

Comment:
Is this change required for the other sdks?

Reply:
Probably so, I'll open a separate PR to make that change everywhere then we can decide whether to merge it.